Lombiq.Hosting.Tenants.Management/Readme.md

```diff
@@ -42,3 +43,23 @@ public void ConfigureServices(IServiceCollection services) =>
 ```
 
 **NOTE:** This extension method not only sets the tags you want to hide but also registers the feature as a setup feature. If you just want to use the default `HideFromSetupScreen` tag then just call the extension method without any parameter.
+
+### `Lombiq.Hosting.Tenants.Management.ShellSettingsEditor`
+
+Adds a shell settings editor to the tenant editor page where you can set values that are not already present in the current ShellSetting for the given tenant. Only those settings will be displayed here that were added from this editor. You can also prefill settings, but it is **not** recommended as it requires to add the same value twice:
+
+```json
+"OrchardCore": {
+  "TenantNamePrefix": {
```

Comment:
Can we instead have a section, like `Editable`? I.e.:

```
  "OrchardCore": {
    "Editable": {
      "TenantName": {
        "TestOption": {
          "TestKey": "TestValue"
        }
      }
    }
  }
```

Then the module would load these values and by default copy them to `OrchardCore__TenantName`. So, no duplication is necessary. This can be done only if the module can hook into config loading even before its `Startup`.

If not, then instead prefix these keys with `Editable_`.

Reply:
This duplication is only "necessary" if you want to preload these settings into this editor, but why would you do that anyway? You can simply use the original not prefixed key in the appsettings, and then override that from the editor, there is really no point in displaying the setting coming from appsettings. This is only for an example.

So do you mean to prefix them like this?
```json
"OrchardCore": {
  "Editable_TenantName": {
    "TestOption": {
      "TestKey": "TestValue"
    }
  },
  {
    "TenantName": {
      "TestOption": {
        "TestKey": "TestValue"
    }
  },
}
```

Reply:
Well, you added this, why would you do that? :) I'm talking about making what's already here better.

Yes, like that.

Reply:
To fulfill this part of the issue description:
> To demonstrate this working, configure [Media storage quota](https://github.com/Lombiq/Hosting-Tenants/blob/dev/Lombiq.Hosting.Tenants.MediaStorageManagement/Readme.md) for a tenant.

But I guess I can do this within a UI test then, so I don't need this at all.

Reply:
Yes.